runtime/metrics.go

```diff
@@ -131,15 +132,41 @@ func (m *Metrics) logSinkJSON(filename string) MetricSinkFn {
 	}
 }
 
+func processTags(customtags []string) map[string]string {
+	m := make(map[string]string)
```

Comment:
Can we alloc the cap that is the length of the incoming tags + length of global tags?

Reply:
I think we can just allocate a static default cap (10 for example) - I really doubt we will use more than 1-2 custom tags, and we have 4 global. WDYT?

Reply:
I've added the preallocation, I guess that's what you had in mind.

Reply:
Yep, perfect!